### PR TITLE
New output feature

### LINKED ISSUE #4: [suggestion] add an output if anything found
Great action, but

I would like to label issues which may be duplicates. Currently it's not easily doable. Could you add an output to this actions that returns for example true if a similar issue was found or false otherwise?

### PR DESCRIPTION
**Pull request by Wxh16144**
**Background**

resolved #4 

**Realization**

Add output function

**Usage**

```yml
name: Issue similarity

on:
  issues:
    types: [opened, reopened, edited]

permissions: read-all

jobs:
  similarity:
    runs-on: ubuntu-latest
    permissions:
      issues: write  # for actions-cool/issues-similarity-analysis to create issue comments
    steps:
      - name: "analysis"
        uses: actions-cool/issues-similarity-analysis@v1
        id: 'analysis-issue'

      - name: "add labels"
        if: steps.analysis-issue.outputs.similar-issues-found == 'true'
        uses: actions-cool/issues-helper@v3
        with:
          actions: 'add-labels'
          labels: '🔥 Similar'
```
The above configuration describes that when an `issues` is created, reopened, or edited, a similar issue determination is made, and the job name `analysis`. When the analysis result has content, add a `🔥 Similar` tag to the issues.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -56,7 +58,6 @@ async function run() {
         return false;
       }
 
-      const result = [];
```
Comment (xrkffgg) [inline src/main.js]: 这个保持不动，报错情况不输出

Comment (Wxh16144) [reply src/main.js]: resolved~

File: src/main.js [hunk 2]
```diff
@@ -84,6 +84,10 @@ async function run() {
       } else {
         await doRemoveIssueComment(owner, repo, number, FIXCOMMENT);
       }
+
+      core.setOutput('similar-issues', JSON.stringify(result));
```
Comment (xrkffgg) [inline src/main.js:88]: 根目录的 action.yml 看下

Comment (Wxh16144) [reply src/main.js:88]: resolved~

**Pull request merged**